sdk/core/azure-core-http-netty/README.md

```diff
@@ -15,7 +15,7 @@ Azure Core Netty HTTP client is a plugin for the azure-core HTTP client API.
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-core-http-netty</artifactId>
-    <version>1.2.0</version>
```

Comment:
Do I undo these non storage ones?

Reply:
Yes, the non-Storage versions in READMEs should be reverted. Run the update command with `--sr` to skip updating README files, or just manually revert the OkHttp and Netty changes.

Reply:
Done!